sources/install/package_mobile.sh

```diff
@@ -90,6 +90,16 @@ function install_androguard() {
     add-to-list "androguard,https://github.com/androguard/androguard,Reverse engineering and analysis of Android applications"
 }
 
+function install_mobsf(){
+    colorecho "Installing Mobile Security Framework"
+    fapt wkhtmltopdf
+    git -C /opt/tools clone --depth 1 https://github.com/MobSF/Mobile-Security-Framework-MobSF
+    python3 -m pipx install /opt/tools/Mobile-Security-Framework-MobSF/
```

Comment:
Please replace `python3 -m pipx` with `pipx` (this allows the commands to be caught by the install process in case some network issue occurs, see the code below to understand)
https://github.com/ThePorgs/Exegol-images/blob/e5bc0af89c30b371102a7bd1c0eee6bd04070969/sources/install/common.sh#L5
https://github.com/ThePorgs/Exegol-images/blob/e5bc0af89c30b371102a7bd1c0eee6bd04070969/sources/install/common.sh#L75-L128

Reply:
done